packages/core/src/components/Dialog/__snapshots__/Dialog.test.jsx.snap

```diff
@@ -10,9 +10,10 @@ Object {
   "wrapper": <Displaced
     dialogClass="ds-c-dialog ds-base"
     escapeExits={true}
+    focusDialog={true}
     includeDefaultStyles={false}
     mounted={true}
-    titleId="dialog-title"
+    titleId="dialog-title dialog-content"
```

Comment:
In my testing, I found that the `dialog-content` ID needed to be on an element without children, otherwise screen readers aren't what or how much to announce. I tested with VoiceOver, and still need to test with JAWS and NVDA, but it seems like a good rule to follow.

So instead of being on the `main` element, it would be on a paragraph inside the `main`, as an example.

That's why I was wondering if it's better to create a separate example with this type of setup?

Reply:
Oh gotcha - what would that look like for the example provided in #631  where there are 2 paragraphs in the `main` element?

Reply:
@line47 sorry, but what is the purpose of the `dialog-content` again? is that needed to work with `focusDialog`?

Reply:
@line47 @davidakennedy If we are using `focusDialog`, would it still be necessary to also change the `titleId` to include both the title and content?

Reply:
Adding to the `titleId ` was to get screen readers to speak the title and content. But the content is variable so maybe it makes more sense to leave it out and add some guidance on how to add it in if desired. 

> Also, you can pass an additional ID via the titleId prop, that way screen readers will associate two labels with it, and usually announce both.

Reply:
> what would that look like for the example provided in #631 where there are 2 paragraphs in the main element?

@line47 In my testing, it would be best to put a wrapper div around the elements in the main container that need to be announced. All three major screen reader/browser combinations announced it that way. When the `dialog-class` is on the `main` element, only JAWS with IE 11 announces it right. Not exactly sure why that is the case.

Should we make a separate example to demo this use case?

Reply:
Making sure I'm understanding here... so folks would need to customize the dialog component to get additional screen reader announcements based on usage of the `titleId` prop and what works best for their scenario?

Reply:
Right. We'd could advise them what to do based on the content of their modal, and what they were after.

Reply:
Ah I see, its a little odd that we need to use `titleId` to also read the content.  I thought normally if you focus a container it will read out everything inside, but since `react-aria-modal` is doing something with `titleId` it might be overriding the default browser behavior maybe. 

Regardless, can we just wrap the `children` with another div with the `dialog-content` className? Something like this:
```
<main role="main">
        <div id="dialog-content">
          {children}
        </div>
</main>
```
It adds an extra level of nesting but it would make it easier on the user

Reply:
I was thinking that example would work if there was only one paragraph in the `{children}` section but if there was more than one paragraph of content addition customization would be needed.

Reply:
So would there be a difference between these 2 code blocks?
 I was thinking these are essentially the same where `id="dialog-content"` is outside of children. and what we Ideally want is to put an id on each child element with a unique id.
```
<main role="main">
        <div id="dialog-content">
          {children}
        </div>
</main>
```
```
<main role="main" id="dialog-content">
          {children}
</main>
```

Reply:
> In my testing, it would be best to put a wrapper div around the elements in the main container that need to be announced. All three major screen reader/browser combinations announced it that way

I just suggested the additional `div` because of what @davidakennedy said. My understanding is that if theres a wrapper with `dialog-content` it wouldn't matter if its 1 or 2 paragraphs.

Reply:
Ah gotcha, I missed that nuance in the earlier thread. I definitely think adding the extra div would be beneficial to set folks up for success
```
<main role="main">
        <div id="dialog-content">
          {children}
        </div>
</main>
```

Reply:
@davidakennedy could you give this another pass?

Reply:
@line47 I tested some more and it's still not working well.

For some reason, both VoiceOver and NVDA don't announce the content in the `main` element, as the second element referenced by `aria-labelledby`. To be honest, not sure why that is, probably just quirks between browsers and screen readers. The way I found that it works across the board is to have it not on a `main`, but another element.

Also, if we go this route,  we could either hide the outline in some browsers on the dialog div or have it be more visually appealing.

Reply:
@line47 Retesting this after your ping in Slack. This works with all major combinations nicely:
- IE 11/JAWS
- Edgium/JAWS
- Chrome/JAWS
- NVDA/Firefox
- VoiceOver/Safari

Still wondering about this from my last comment:
> Also, if we go this route, we could either hide the outline in some browsers on the dialog div or have it be more visually appealing.